tests/configuration/verifyPMMSettingsPageFunctionality_test.js

```diff
@@ -348,6 +349,25 @@ Scenario(
     scheduledPage.openScheduledBackupsPage();
   },
 );
+
+Scenario(
+  'PMM-T1658-backup-managment-enabled-by-default @backup',
+  async ({
+    I, pmmSettingsPage, settingsAPI, homePage,
+  }) => {
+    const settingEndpointResponse = await settingsAPI.getSettings('backup_management_enabled');
+    const bmIcon = locate('a[type="button"][aria-label="Backup"]');
```

Comment:
can you move this locator to the page object?

Reply:
There is no POM for the left menu at least I am not aware of it this solution was decided after a discussion with @yurkovychv